src/main/java/com/codessquad/qna/domain/Question.java

```diff
@@ -1,22 +1,45 @@
 package com.codessquad.qna.domain;
 
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.Id;
+import org.hibernate.annotations.CreationTimestamp;
+
+import javax.persistence.*;
 import java.time.LocalDate;
+import java.util.Objects;
+
 @Entity
 public class Question {
 
     @Id
     @GeneratedValue
     private Long id;
 
-    private String writer;
     private String title;
     private String content;
     private int index;
+    @CreationTimestamp
     private LocalDate nowDate;
```

Comment:
```suggestion
    private int index;
    
    @CreationTimestamp
    private LocalDate nowDate;
```
이런 식으로 애노테이션이 붙은 필드는 뗴어주시고, `nowDate`라는 말은 무조건 현재 시간이어야 할 것 같은데, 다른 이름을 짓는것이 어떤가요?

Reply:
nowDate -> time으로 필드명 수정 했습니다!